libsmile.mk

```diff
@@ -36,7 +36,7 @@ LIBTOOL  = libtool
 endif
 
 SMILE_INCLUDE = -I$(CURDIR)/include
-CFLAGS   += -std=c99 -pedantic -Wall -Wmissing-prototypes -Wstrict-prototypes -O2 -fomit-frame-pointer -I. $(SMILE_INCLUDE)
+CFLAGS   += -std=c99 -pedantic -Wall -Wmissing-prototypes -Wstrict-prototypes -O2 -fomit-frame-pointer -I. $(SMILE_INCLUDE) -g
```

Comment:
This is optional, it puts debug symbols into both `libsmile.a` and `unsmile`... which let me gdb it more easily when it was segfaulting.